scipy/stats/_continuous_distns.py

```diff
@@ -3569,34 +3569,61 @@ def fit(self, data, *args, **kwds):
         data, floc, fscale = _check_fit_input_parameters(self, data,
                                                          args, kwds)
 
-        # if user has provided `floc` or `fscale`, fall back on super fit
-        # method. This scenario is not suitable for solving a system of
-        # equations
-        if floc is not None or fscale is not None:
-            return super().fit(data, *args, **kwds)
-
-        # rv_continuous provided guesses
-        loc, scale = self._fitstart(data)
-        # account for user provided guesses
-        loc = kwds.pop('loc', loc)
-        scale = kwds.pop('scale', scale)
-
         # By the method of maximum likelihood, the estimators of the
         # location and scale are the roots of the equation defined in
         # `func` and the value of the expression for `loc` that follows.
         # Source: Statistical Distributions, 3rd Edition. Evans, Hastings,
         # and Peacock (2000), Page 101
```

Comment:
I'm thinking about adding another sentence here to address where this equation (not from book)
```
                def func(scale):
                    return (((floc - data) * np.exp(
                        (floc - data) / scale) + data).sum() -
                            len(data) * (floc + scale))
```
is from.

This comment only addressed the `func` from the book so it still needs to be updated.